meson.build

```diff
@@ -300,7 +301,14 @@ else
 endif
 
 if want_libfuzzer
-        fuzzing_engine = meson.get_compiler('cpp').find_library('Fuzzer')
+        fuzzing_engine = meson.get_compiler('cpp').find_library('Fuzzer', required : false)
+        if fuzzing_engine.found()
+                add_project_arguments('-fsanitize-coverage=trace-pc-guard,trace-cmp', language : 'c')
```

Comment:
Why are you using trace-pc-guard instrumentation with libFuzzer? It's deprecated, you should be using fuzzer-no-link.

Reply:
The "want_libfuzzer" mode is supposed to also work on systems where `fuzzer-no-link` isn't supported. Last time I checked clang-3.9, clang-4.0 and clang-5.0 bailed out receiving `-fsanitize=fuzzer-no-link`. And, as far as I know, `libFuzzer` isn't shipped as a library with clang-6+.

Reply:
Ah OK, yeah it should still work with old versions of LF.

>And, as far as I know, libFuzzer isn't shipped as a library with clang-6+.

What do you mean by this?

Reply:
> What do you mean by this?

While I was answering the question about why I use `trace-pc-guard` I realized that I can add another `if` to check whether `fuzzer-no-link` is supported but then remembered that it's very unlikely that `libFuzzer` is installed and `fuzzer-no-link` is supported at the same time.